samples/Calling/src/app/App.tsx

```diff
@@ -23,6 +24,8 @@ import { EndCall } from './views/EndCall';
 import { HomeScreen } from './views/HomeScreen';
 import { UnsupportedBrowserPage } from './views/UnsupportedBrowserPage';
 
+setLogLevel('warning');
```

Comment:
Is this useful to include in the server also? (or just adding it for UFD related warnings)

Reply:
Doesn't make sense on Server. We already use a different logger there. Azure logger is for browser logs primarily.
The good thing is, we might be able to easily send these logs elsewhere (like somewhere on the cloud) if we use this logger.